test/e2e/testnet/node.go

```diff
@@ -294,14 +298,26 @@ func (n Node) Client() (*http.HTTP, error) {
 }
 
 func (n *Node) Start() error {
-	if err := n.Instance.Start(); err != nil {
+	if err := n.StartAsync(); err != nil {
+		return err
+	}
+	if err := n.WaitUntilStartedAndForwardPorts(); err != nil {
```

Comment:
no more proxy instead of port forward? or that will come later?

Reply:
Yeah good point. What order should this PR be merged in respect to the proxying ports PR